india_compliance/gst_india/doctype/purchase_reconciliation_tool/purchase_reconciliation_tool.js

```diff
@@ -1174,6 +1182,240 @@ class ImportDialog {
     }
 }
 
+class ExportData {
```

Comment:
Data processing should be avoided in frontend (as it is RAM Intensive).
For Reco tool, we made exception for 2 things, Filtering and Summarizing as everything is handled in front end.
Here when we are having reliance on backend, we should have as much reliance as possible.

You can move this complete class to python, and differentiate it there.